modules/@apostrophecms/image-widget/index.js

```diff
@@ -10,10 +10,33 @@ module.exports = {
     add: {
       _image: {
         type: 'relationship',
+        editor: 'AposImageRelationshipEditor',
         label: 'apostrophe:image',
         max: 1,
         required: true,
-        withType: '@apostrophecms/image'
+        withType: '@apostrophecms/image',
+        fields: {
+          add: {
+            top: {
+              type: 'integer'
+            },
+            left: {
+              type: 'integer'
+            },
+            width: {
+              type: 'integer'
+            },
+            height: {
+              type: 'integer'
+            },
+            x: {
+              type: 'integer'
+            },
+            y: {
+              type: 'integer'
+            }
```

Comment:
```suggestion
            top: {
              type: 'integer',
              label: 'top'
            },
            left: {
              type: 'integer',
              label: 'left'
            },
            width: {
              type: 'integer',
              label: 'width'
            },
            height: {
              type: 'integer',
              label: 'height'
            },
            x: {
              type: 'integer',
              label: 'x'
            },
            y: {
              type: 'integer',
              label: 'y'
            }
```

Reply:
Will add label when working on the modal :)